frontend/src/components/workspace/CreateApplicationButton.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { startApplication } from "src/services/fetch/fetchers/clientApplicationFetcher";
+
+import { useRouter } from "next/navigation";
+import { useCallback, useState } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+const CreateApplicatinButton = () => {
+  const router = useRouter();
+  const [loading, setLoading] = useState<boolean>();
+
+  const handleSubmit = useCallback(() => {
+    setLoading(true);
+    startApplication("insecuretoken")
+      .then((data) => {
+        const { applicationId } = data;
+        router.push(`/formPrototype/${applicationId}`);
+      })
+      .catch((error) => {
+        console.error(error);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, [router]);
+  return (
+    <Button onClick={handleSubmit} type="submit">
+      {loading ? "loading..." : "Create new application"}
```

Comment:
are you planning to have a ticket to go back through and put all the strings through translation later?

Reply:
That is out of scope for the demo. Once design provides a design we can add the translations.